docs/HowtoTDE.md

```diff
@@ -0,0 +1,219 @@
+Starting from NuoDB v4.1.2 and NuoDB Helm Charts v3.1.0 Transparent Data Encryption is supported in NuoDB Kubernetes deployments.
+
+Enabling TDE consists of two steps
+1. [Supply storage passwords](#supplying-storage-passwords)
+2. [Configure database layer](#enable-transparent-data-encryption)
+
+> **NOTE**: For information on enabling TDE in non-Kubernetes deployments of NuoDB, see [here](https://doc.nuodb.com/nuodb/latest/database-administration/transparent-data-encryption/configuring-transparent-data-encryption/). 
+This document expands on the product documentation and is specific to this Helm Chart repository.
+
+### Terminology
+
+- `NuoDB Admin` = admin interface for domain and database management. Started by the [Admin Chart](../stable/admin/README.md).
+- `Storage password` = key used by database engines to secure the encrypted data.
+- `Target password` = Password that needs to be set as _current_ database storage password.
+- `Historical password`= Password used to encrypt _any_ archive in the database or a backupset. 
+There could be multiple historical passwords.
+- `TDE monitor service` = NuoDB Admin auxiliary service which can read storage passwords configuration from files on disk.
+
+
+## Supplying storage passwords 
+
+The domain administrator is responsible for configuring TDE storage passwords. They can be supplied as Kubernetes secret or injected inside the pods using 3rd party software like HashiCorp Vault.
+If using third-party party software, the storage passwords need to be injected and stored as files inside AP and SM pods. NuoDB supports file per password approach explained in [using Kubernetes secrets](#using-kubernetes-secrets) and single configuration file per database approach explained in [HashiCorp Vault integration](#using-hashicorp-vault).
+Choosing the right approach depends on the capabilities and flexibility that any other 3rd party tool securing the storage passwords have.
+
+> **IMPORTANT**: NuoDB, Inc. cannot recover Transparent Data Encryption storage passwords, and the database cannot be loaded if its storage password is lost. It is the user’s responsibility to keep track of storage passwords, including historical passwords for encrypted hot copy backups.
+
+### Using Kubernetes secrets
+
+Storage passwords can be supplied using Kubernetes secret.
+One secret per database should be created in the Kubernetes namespace where NuoDB is deployed.
+It must hold the `target` and optional `historical` storage passwords for that database.
+The configured secret name is mounted inside all AP, hotcopy SM and non-hotcopy SM pods. The mount path is `/etc/nuodb/tde/<db-name>` by default and the static portion of the path can be changed using `admin.tde.storagePasswordsDir` variable.
+NuoDB Admin monitors all directories and files under `/etc/nuodb/tde` path and will configure storage passwords available in the mounted files. The following files can be used:
```

Comment:
suggest, "... user **the** `/etc/nuodb/tde` path ..."
suggest, "... configure **the** storage ..."